### DISCUSSION TITLE
`reset()` is missing in `solito/router`

### DISCUSSION THREAD

**Discussion by saxenanickk** [nandorojo/solito#417] [Ideas]
this is must have option for a lot of mobile applications where you need to [reset the navigator](https://reactnavigation.org/docs/navigation-actions#reset).

@nandorojo  we should have this.

**Comment by nandorojo** [comment]
you can use react navigation for this on the native side. there’s no web equivalent. i’ve also never used it myself

**Reply by a-eid** [reply]
@nandorojo how do you perform logout on native apps ?

**Reply by GoDandreaIrya** [reply]
I'm having this very problem right now, actually. The user can log out but state isn't reset and components aren't unmounted. 
Reset seems to be the main way of implementing the desired behavior.
